app/assets/javascripts/channels/messages.js

```diff
@@ -1,9 +1,15 @@
-App.messages = App.cable.subscriptions.create('MessagesChannel', {  
+$(document).ready(function() {
+chatroomId = $('input#message_chatroom_id').val();
+App.messages = App.cable.subscriptions.create({channel: 'MessagesChannel', chatroom_id: chatroomId}, {  
   received: function(data) {
-    $("#messages").removeClass('hidden')
+    $("#messages").removeClass('hidden');
     return $('#messages').append(this.renderMessage(data));
   },
+  chatroom_id: function(data) {
```

Comment:
Don't think we need this `chatroom_id` function?

Reply:
I wrote this code a long time ago but I believe it's needed to identify the chatroom.